SPECS/pyparsing/pyparsing.spec

```diff
@@ -1,17 +1,18 @@
 Summary:        Python package with an object-oriented approach to text processing
 Name:           pyparsing
-Version:        3.0.7
+Version:        3.1.1
 Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Languages/Python
 URL:            https://github.com/pyparsing/pyparsing
-Source0:        https://github.com/pyparsing/pyparsing/releases/download/%{name}_%{version}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/pyparsing/pyparsing/releases/download/%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  python3-devel
-%if %{with_check}
```

Comment:
Are all of these BuildRequires needed even when not running tests?

Reply:
Good point! Yes, I verified that those are required for the new build and install steps.